src/jdk.security.jgss/share/classes/com/sun/security/sasl/gsskerb/GssKrb5Client.java

```diff
@@ -150,6 +151,14 @@
             }
             secCtx.requestMutualAuth(mutual);
 
+            if (props != null) {
+                // TLS Channel Binding
+                byte[] tlsCB = (byte[])props.get("jdk.internal.sasl.tlschannelbinding");
```

Comment:
You can say the name is defined in another class in another module. If we really want to rename it one day we will know where it's from.

Reply:
Thank you. Comment is added